### PR TITLE
sssctl: sssctl analyze command added

### PR DESCRIPTION
**Pull request by patriki01**
Ask, if something is not clear.

### CODE REVIEW

File: sssd_test_framework/utils/sssctl.py [hunk 1]
```diff
@@ -263,3 +263,26 @@ def domain_status(
         }
 
         return self.host.ssh.exec(["sssctl", "domain-status"] + self.cli.args(args), raise_on_error=False)
+
+    def analyze(
```
Comment (justin-stephenson) [inline sssd_test_framework/utils/sssctl.py:267]: could be better to call this `analyze_request` and always execute request subcommands? We have an analyze `error` module (https://github.com/SSSD/sssd/commit/ed3726c37fe07aab788404bfa2f9003db15f4210) but no need to test it at the moment

Comment (patriki01) [reply sssd_test_framework/utils/sssctl.py]: Good point. Fixed

**Review by justin-stephenson — APPROVED** [review]
Ack thank you.

**Review by thalman — APPROVED** [review]
ACK

**Pull request merged**